data/raw/dap.ceda.ac.uk/README.md

```diff
@@ -0,0 +1,12 @@
+The data in this directory can be downloaded from https://catalogue.ceda.ac.uk/uuid/ae4f1eb6fce24adcb92ddca1a7838a5c.
+
+Alternately, the following script will download the data if `wget` is installed.
+
+```bash
+#!/bin/bash
+OUTPUT_DIRECTORY="./"
+
+wget -e robots=off --mirror --no-parent -r https://dap.ceda.ac.uk/badc/ar6_wg1/data/spm/spm_01/v20210809/ --directory-prefix="${OUTPUT_DIRECTORY}"
```

Comment:
If this is taken from the `config.yaml`, wouldn't it make more sense to refer to that file? Otherwise, we risk getting out of sync between the explicit list and the config...

Reply:
Ok I made it more explicit. I agree it's a risk of getting out of sync. I think the only way to properly mitigate that risk would be to add CI so I made #4.